scripts/install

```diff
@@ -106,10 +105,19 @@ ynh_use_logrotate
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..." --weight=1
 
-yunohost service add "$app" --description="Jellyfin media center"
+ports_args=()
+if [[ "${opened_ports:-__NOTHING__}" != "__NOTHING__" ]]; then
```

Comment:
FYI you can also use `-z` to check the emptiness of the string. Your code is completely fine with me though, don't change it.